test/functorch/test_vmap.py

```diff
@@ -3361,11 +3361,45 @@ def test():
                             vmap(op, in_dims)(*args, **kwargs)
 
             # Sample inputs check
-            sample_inputs_itr = op.sample_inputs(device, dtype, requires_grad=False)
+            sample_inputs_op = {
+                                # Take too long
+                                "special.chebyshev_polynomial_t",
```

Comment:
These ops already have skip for taking long time with reference inputs

Eg.
https://github.com/pytorch/pytorch/blob/39d49dbe4546c9f4d153ec748f62371e4546158a/torch/testing/_internal/opinfo/definitions/special.py#L374-L388